microsoft-edge/devtools-guide-chromium/whats-new/whats-new-archive.md

```diff
@@ -0,0 +1,34 @@
+---
+title: What's new in Microsoft Edge DevTools Archives
+description: New features and fixes in Microsoft Edge DevTools Archives
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 01/23/2023
+---
+# Archived What's New entries
+
+This page is an archive for all of the legacy [What's New in DevTools](whats-new.md) entries published after each release.
+
+<!-- adjust placement and indent in toc.yml too -->
+
+* [What's New in DevTools (Microsoft Edge 99)](2022/03/devtools.md)
+* [What's New in DevTools (Microsoft Edge 98)](2022/02/devtools.md)
+* [What's New in DevTools (Microsoft Edge 97)](2022/01/devtools.md)
+* [What's New in DevTools (Microsoft Edge 96)](2021/11/devtools.md)
+* [What's New in DevTools (Microsoft Edge 95)](2021/10/devtools.md)
+* [What's New in DevTools (Microsoft Edge 94)](2021/09/devtools.md)
+* [What's New in DevTools (Microsoft Edge 93)](2021/07/devtools.md)
+* [What's New in DevTools (Microsoft Edge 92)](2021/05/devtools.md)
+* [What's New in DevTools (Microsoft Edge 91)](2021/04/devtools.md)
+* [What's New in DevTools (Microsoft Edge 90)](2021/02/devtools.md)
+* [What's New in DevTools (Microsoft Edge 89)](2021/01/devtools.md)
+* [What's New in DevTools (Microsoft Edge 88)](2020/11/devtools.md)
+* [What's New in DevTools (Microsoft Edge 87)](2020/10/devtools.md)
+* [What's New in DevTools (Microsoft Edge 86)](2020/08/devtools.md)
+* [What's New in DevTools (Microsoft Edge 85)](2020/06/devtools.md)
+* [What's New in DevTools (Microsoft Edge 84)](2020/05/devtools.md)
+* [What's New in DevTools (Microsoft Edge 83)](2020/03/devtools.md)
+* [What's New in DevTools (Microsoft Edge 81)](2020/01/devtools.md)
+* [What's New in DevTools (Microsoft Edge 80)](2019/12/devtools.md)
```

Comment:
un-link the intro section and use instead one of the other standard format options:

Prefer link pattern using one of these options:

For more information see [link]
(this would be in paragraph 1 at top of page)

See also:
* [link]
(this would be below the list of links)

h2 See also 
* [link]

(this would be below the list of links)